### ISSUE TITLE
Create pivottable based on multiple tables (data model)

### ISSUE THREAD

**Issue by rpresser** [EPPlusSoftware/EPPlus#256]
Is there any support for doing this?   All the constructors seem to take a single table (or range). See [my StackOverflow question](https://stackoverflow.com/questions/65222280/relationships-between-multiple-tables-in-eppplus) for more detail. Also see  attached [test.xlsx](https://github.com/EPPlusSoftware/EPPlus/files/5681806/test.xlsx) that demonstrates the kind of pivottable I mean.

**Comment by JanKallman** [comment]
No, there is not support for data models at this point. I'll add this as an enhancement for future versions.
